client/src/components/menubar/index.tsx

```diff
@@ -330,21 +330,19 @@ class MenuBar extends React.PureComponent<{}, State> {
             justifyContent: "right",
           }}
         >
-          {seamlessEnabled ? (
```

Comment:
Ah great catch! Is the reason because dataset drawer would still have information from the dataset itself that's being displayed in the drawer, which doesn't depend on Data Portal APIs?

Reply:
If so, please help add a comment in the code, so we have the context in the future 🙏 Thank you!

Reply:
I went to comment it in the infodrawer files and its already documented